doc/conf.py

```diff
@@ -102,8 +102,8 @@
 # Output file base name for HTML help builder.
 htmlhelp_basename = 'CabalUsersGuide'
 
-# MathJax to use SVG rendering by default
-mathjax_path = 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-AMS-MML_SVG'
+# MathJax to use HTML rendering by default (makes the text selectableable, see #8453)
```

Comment:
```suggestion
# MathJax to use HTML rendering by default (makes the text selectable, see #8453)
```

Reply:
Oh, thank you!How hard could it bee to not slip in one sentence...